scalacheck/src/main/scala/pathy/scalacheck/PathOf.scala

```diff
@@ -54,19 +54,14 @@ object PathOf {
     genRelFile[A] map (rootDir </> _)
 
   private def genRelFile[A: Arbitrary: Show]: Gen[RelFile[Sandboxed]] =
-    for {
-      d <- genRelDir[A]
-      s <- genSegment[A]
-    } yield d </> file(s)
+    sizeDistributed(genRelDir[A], genSegment[A])((d, s) => d </> file(s))
```

Comment:
So here if the `size` parameter is set to 10, in the old version, we would get a path with 10 directories with 10 characters each and a filename of 10 characters?